Server/MirEnvir/Envir.cs

```diff
@@ -19,7 +19,7 @@ public class Envir
         public static object AccountLock = new object();
         public static object LoadLock = new object();
 
-        public const int Version = 53;
+        public const int Version = 55;
```

Comment:
why was the version increases for this request? I don't see anything that needs it